ocs_ci/utility/environment_check.py

```diff
@@ -85,73 +118,58 @@ def get_status_after_execution():
     """
     get_environment_status(ENV_STATUS_POST)
 
-    pod_diff = DeepDiff(
+    pod_diff = compare_dicts(
         ENV_STATUS_PRE['pod'], ENV_STATUS_POST['pod']
     )
-    sc_diff = DeepDiff(
+    sc_diff = compare_dicts(
         ENV_STATUS_PRE['sc'], ENV_STATUS_POST['sc']
     )
-    cephfs_diff = DeepDiff(
+    cephfs_diff = compare_dicts(
         ENV_STATUS_PRE['cephfs'], ENV_STATUS_POST['cephfs']
     )
-    cephbp_diff = DeepDiff(
+    cephbp_diff = compare_dicts(
         ENV_STATUS_PRE['cephbp'], ENV_STATUS_POST['cephbp']
     )
-    pv_diff = DeepDiff(
+    pv_diff = compare_dicts(
         ENV_STATUS_PRE['pv'], ENV_STATUS_POST['pv']
     )
-    pvc_diff = DeepDiff(
+    pvc_diff = compare_dicts(
         ENV_STATUS_PRE['pvc'], ENV_STATUS_POST['pvc']
     )
-    secret_diff = DeepDiff(
+    secret_diff = compare_dicts(
         ENV_STATUS_PRE['secret'], ENV_STATUS_POST['secret']
     )
-    namespace_diff = DeepDiff(
+    namespace_diff = compare_dicts(
         ENV_STATUS_PRE['namespace'], ENV_STATUS_POST['namespace']
     )
     diffs_dict = {
         'pods': pod_diff,
-        'sc': sc_diff,
+        'storageClasses': sc_diff,
         'cephfs': cephfs_diff,
         'cephbp': cephbp_diff,
```

Comment:
Isn't this cephblockpool?

Reply:
cephblockpool is 2 lines below (cephbp).
It is just a name for the resource